### ISSUE TITLE
Could you please provide non-playstore apk download link?

### ISSUE THREAD

**Issue by dotaheor** [tailscale/tailscale#762]
For some reason, I can't visit Google play store.
So it would be great if you can put the apk file somewhere else, such as github.com.

There are many free apk download websites, but I don't trust them.

**Comment by bradfitz** [comment]
Can you run APKs that depend on Google Play Services?

We can host them on pkgs.tailscale.com but probably once we move to doing the Android builds out of our umbrella git repo that git-submodules in all the other repos. (the umbrella repo is what controls the version numbers, and it'd be nice to get the Android client's version numbers in sync with everything else)

/cc @eliasnaur @danderson

**Comment by dotaheor** [comment]
Does Tailscale need Play services?

I just realized this issue is not very valid, for I plan to use Google GSuite to auth my phone.
I installed wireguard one my phone, it worked well for a long time, but suddenly failed to work since some point.
I haven't figured out the reason why it doesn't work now. 
When it worked, I can use play store to install apps and use Play services.

So now even if I could install Tailscale, I may still not auth my phone with Google GSuite.

So feel free to close this issue.
